gradle-dependency-check/action.yml

```
@@ -0,0 +1,54 @@
+name: Run a Gradle Dependency Check scan
+description: |
+  Run a Gradle Dependency Check scan on the project.
+  This action requires the global-conventions  plugin to be installed.
+  See https://github.com/kronostechnologies/standards/tree/master/gradle/global-conventions
+
+inputs:
+  gradle-properties:
+    description: Content of a gradle.properties file that will be passed to the gradle runner.
+    required: false
+  working-directory:
+    description: Relative path under $GITHUB_WORKSPACE where the root project is located.
+    required: false
+    default: "."
+  report-name:
+    description: The name of the archived report.
+    required: true
+    default: dependency-check-report
+  report-retention-days:
+    description: Duration in days to preserve reports.
+    required: true
+    default: "5"
+  task-name:
+    description: The DependencyCheck task name. Defaults to "dependencyCheckAggregate".
+    required: true
+    default: dependencyCheckAggregate
+
+runs:
+  using: composite
+  steps:
+    - name: Scan Gradle dependencies
+      uses: burrunan/gradle-cache-action@v1.10
+      with:
+        gradle-version: wrapper
+        build-root-directory: ${{ inputs.working-directory }}
+        gradle-dependencies-cache-key: |
+          buildSrc/**/Dependencies.kt
+        arguments: ${{ inputs.task-name }}
+        properties: ${{ inputs.gradle-properties }}
+
+    - name: Upload HTML report
+      uses: actions/upload-artifact@v2
+      if: github.action_status != 'cancelled'
```

J'ai dû remplacer par les fonctions built-in. Utiliser le contexte ne semble pas marcher pour les `if`

```suggestion
      if: "!cancelled()"
```